CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 BREAKING CHANGES:
 * Update minimum go version for project to 1.17 [[GH-878](https://github.com/hashicorp/consul-k8s/pull/878)]
 
+FEATURES:
+* CLI
+   * Add `upgrade` command to modify Consul installation on Kubernetes. [[GH-898](https://github.com/hashicorp/consul-k8s/pull/898)]
```

Comment:
Can we mark this as beta here since we haven't got automated acceptance tests yet?